src/app/products/components/product-delete-confirm-dialog/product-delete-confirm-dialog.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>Please confirm</h1>
+<div mat-dialog-content>
+    <p>Are you sure you want to delete {{ product.name }}?</p> 
+</div>
+<div mat-dialog-actions class="dialog-actions">
+    <button mat-button (click)="onCancelClick()">Cancel</button>
+    <button mat-button [mat-dialog-close]="product" cdkFocusInitial>Delete</button>
```

Comment:
Order of element attributes, properties, and directives
When adding attributes and properties to an element, the following order must be followed.

1. Structural Directives
2. Template Reference Variable
3. Element ID
4. Element Class
5. Conditional classes
6. Non-structural directives, attributes and input properties without brackets
7. Inputs with brackets
8. Bi-directional bindings
9. Outputs

Example:
`<div *ngIf="shouldShow"
         #myRefName
         id="my-id"
         class="some-class"
        [class.conditional-class]="someCondition"
        [someInput]="true"
        (someOutput)="someMethod($event)">
</div>`